tests/ops/qutrit/test_qutrit_channel_ops.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the available built-in qutrit quantum channels.
+"""
+import pytest
+import numpy as np
+from numpy.linalg import matrix_power
+import pennylane as qml
+from pennylane import numpy as pnp
+from pennylane.ops.qutrit import channel
+
+QUDIT_DIM = 3
+
+
+class TestQutritDepolarizingChannel:
+    """Tests for the qutrit quantum channel QutritDepolarizingChannel"""
+
+    @staticmethod
+    def get_expected_kraus_matrices(p):
+        """Gets the expected Kraus matrices given probability p."""
+        w = np.exp(2j * np.pi / 3)
+
+        X = np.array([[0, 1, 0], [0, 0, 1], [1, 0, 0]])
+        Z = np.diag([1, w, w**2])
+
+        Ks = [np.sqrt(1 - p) * np.eye(QUDIT_DIM)]
+        for i in range(3):
```

Comment:
Consider using [itertools.product](https://docs.python.org/3/library/itertools.html#itertools.product) for nested loops like this.

Reply:
Let's leave as-is then.